doc/api-extensions.md

```diff
@@ -240,6 +240,13 @@ will look for an existing interface for that VLAN on the host. If one
 can't be found it will create one itself and then use that as the
 macvlan parent.
 
+## network\_gvrp
```

Comment:
@tomponline this should go to the bottom of the file

Reply:
@stgraber ah yes indeed, I hadn't noticed that it was added in the middle of the file.

Reply:
@stgraber fixed